src/js/components/ExpandableGroup/ExpandableGroup.tsx

```diff
@@ -110,6 +112,10 @@ const ExpandableGroup = ({
           </div>
         </div>
 
+        {!expanded && closedHelperText && (
+          <div className="u-size4of12 u-flex u-flexJustifyEnd">{closedHelperText}</div>
```

Comment:
This should have some shade of grey for the text. Regarding what shade, maybe the mock has the shade or you can ask Isa